app/tag-management/edit-tag/edit-tag.component.css

```diff
@@ -0,0 +1,104 @@
+h3 {
+  font-size: 22px;
+  font-weight: 300;
+  color: #555;
+  line-height: 30px;
+}
+
+img {
+  max-width: 100%;
+}
+
+::-moz-selection {
+  background: #de615e;
+  color: #fff;
+  text-shadow: none;
+}
+
+::selection {
+  background: #de615e;
+  color: #fff;
+  text-shadow: none;
+}
+
+.right {
+  position: relative;
+  width: 100%;
+}
+
+.right button {
+  position: absolute;
+  right: 0;
+  size: 100%;
+}
+
+md-card {
```

Comment:
We already have a class to center the md-card, please use that: `default-card` instead of doing it component specific.